src/presign/round_three.rs

```diff
@@ -63,44 +64,42 @@ pub(crate) struct Public {
 }
 
 impl Public {
+    /// Verify the validity of [`Public`] against the sender's [`AuxInfoPublic`]
+    /// and [`PublicBroadcast`](crate::presign::round_one::PublicBroadcast)
+    /// values.
+    ///
+    /// Note: The `receiver_...` values must be those of the _caller_ (i.e., the
```

Comment:
I can also go ahead and rename these as part of this PR (that is, go from `receiver_...` -> `verifier_...` and `sender_...` -> `prover_...`)